Do not blow away existing attributeBindings

thoov opened the pull request.

runspired commented:
Thanks! These are good changes/fixes :) One question, is `this.applyStyles` something you intend to be accessible for toggling? If so, we should probably use `get` on it when calculating the style string.

runspired commented:
@thoov a change that could slide in here or a separate PR, for perf reasons I think the `htmlSafe` string should be a singleton;  Basically this:

```
const SafeTouchAction = htmlSafe('touch-action: manipulation; -ms-touch-action: manipulation; cursor: pointer;');
const SafeEmptyString = htmlSafe('');

...

return applyStyle ? SafeTouchAction : SafeEmptyString;
```

thoov commented:
@runspired I updated the PR to address your comments. I removed the applyStyles prop as I was simply coping it over from [this pr](https://github.com/runspired/ember-hammertime/pull/18/files). Let me know if you have any other comments
